src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const perDay = 40;
+  const minNumberOfDays = 3;
+  const minDiscount = 20;
+  const maxNumberOfDays = 7;
+  const maxDiscount = 50;
+
+  if (days === 0 || days === '') {
+    return 0;
+  }
+
+  if (days < minNumberOfDays) {
+    return days * perDay;
```

Comment:
Good job!
Just change one thing.
make a separate variable for "days * perDay", you used it three times